src/pathfinding_igvc/src/PathFinderNode.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Created By: Min Gyo Kim
+ * Created On: May 23rd 2018
+ * Description: Implementation of Path Finder Node
```

Comment:
It might be helpful for people in the future if we could make this a bit more descriptive